CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Qiita changelog
 
+Version 112019
```

Comment:
I'm not sure I understand.  Is 112019 the latest or not?  I see 112019 as the current top of the changelog in biocore/qiita/master.

Reply:
What I mean is that since this block is in the latest version of master, the diff shouldn't show these lines as new additions.

I suspect either the `dev` branch in `biocore/qiita` is not up to date (with master), or that your branch is somehow out of date.

Reply:
Both this and the redbiom changes, as well as several other changes that I suspect snuck in as well are authored in commit 9c2e0476641befcac6a8a00d1f3d975dd482e2b4.  I'm not sure why this commit is considered part of this pull request.  As far as I can tell, this commit has already been applied to the biocore/qiita/dev branch.  Presumably this means I busted the git history while I was cleaning up the commit log for the PR...

Reply:
Hmmm, maybe try git pull upstream dev. Otherwise that should be fine.

Reply:
git pull upstream dev says there are no changes.  

I suspect this is the problem mentioned in: https://stackoverflow.com/questions/16306012/github-pull-request-showing-commits-that-are-already-in-target-branch

After the flake8 changes, which took a lot of commits and pushes to resolve, I squashed a series of commits down.  I suspect the github diff procedure is unable to recognize that commit 9c2e047 was applied to both the dev branch and the ST_DH_artifact_download branch, but from what I've read, we can fix by either reopening the pull request or just leaving it be as git will handle the merge correctly.

Reply:
Possibly this is the same issue as: https://stackoverflow.com/questions/16306012/github-pull-request-showing-commits-that-are-already-in-target-branch

Seems like it happens after a rebase, which I did to squash the commits after making the flake8 changes.  Could be fixed by reopening the pull request, or just ignoring it and letting git handle it, since git seems to handle the diff correctly even though github can't.

Reply:
Sounds good, lets leave this aside then.